api/v1beta1/hyperconverged_types.go

```diff
@@ -338,6 +338,7 @@ type HyperConvergedWorkloadUpdateStrategy struct {
 // +k8s:openapi-gen=true
 type HyperConvergedStatus struct {
 	// Conditions describes the state of the HyperConverged resource.
+	// +listType=map
```

Comment:
`openapi-gen` to create theOpenAPIDefinitions code was failing because slice types didn't have a `listType` defined for conversion